CaesarCipher/CaesarCipher/main.m

```diff
@@ -50,11 +44,38 @@ - (NSString *)decode:(NSString *)string offset:(int)offset {
     return [self encode:string offset: (26 - offset)];
 }
 
+- (BOOL)codeBreaker: (NSString *)string word:(NSString*)string2{
+    //Method 1
+//    for (int i=1; i<=25; i++){
+//        for(int j=1;j<=25;j++){
+//            if([[self decode:string offset:i] isEqualToString:[self decode:string2 offset:j]]){
+//                return 1;
+//            }
+//        }
+//    }
+    //Method 2 thanks to V's suggestion
+    for(int i=1;i<25;i++){
+        if([string isEqualToString:[self encode:string2 offset:i]]){
+            return 1;
+        }
+    }
+    return 0;
+}
```

Comment:
I recommend getting used to YES and NO as the two bool values. 1 and 0 is fine, but not the objective-c way.